modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-labels-and-tags_{context}"]
+= Configuring the user-defined labels and tags for GCP
+
+
+Google Cloud Platform (GCP) provides `labels` and `tags` that help to identify and organize the resources created for a specific {product-title} cluster, making them easier to manage. 
+
+You can define `labels` and `tags` for each GCP resource only during {product-title} cluster installation. 
+
+[IMPORTANT]
+====
+User-defined `labels` and `tags` are not supported for {product-title} clusters upgraded to {product-title} 4.14 version.
+====
+
+.User-defined labels
+
+User-defined `labels` and {product-title} specific labels are applied only to resources created by {product-title} installer and its core operators such as: 
+* `machine-api-provider-gcp` 
+* `cluster-image-registry-operator`
+* `gcp-filestore-csi-driver-operator`
+* `gcp-pd-csi-driver-operator` 
+
+The user-defined `labels` and {product-title} specific labels are not applied on the resources created by any other operators or the kubernetes in-tree components that create resources, for example, the Ingress load balancers.
+
+The following are the requirements for the user-defined labels:
+
+* A label key and value must have a minimum of 1 character and can have a maximum of 63 characters.
+* A label key and value must contain only lowercase letters, numeric characters, underscore (`_`), and dash (`-`).
+* A label key must start with a lowercase letter.
+* You can configure a maximum of 32 labels per resource. Each resource can have a maximum of 64 labels, and 32 labels are reserved for internal use by {product-title}.
+
+.Limitations for user-defined labels
+
+* Labels for `Compute Address` are supported in the GCP beta version. {product-title} does not add labels to the resource.
+
+User-defined labels and {product-title} labels are seen on the following GCP resources:
```

Comment:
Are they seen? there could be better wordings like `available`.

Reply:
Since what SME meant was the labels would be found added to the GCP resources, I went for the easier alternative.

Reply:
changed.